snappy_pipeline/workflows/mobile_element_insertion/__init__.py

```diff
@@ -0,0 +1,360 @@
+# -*- coding: utf-8 -*
+"""Implementation of the ``mobile_element_insertion`` step
+
+The ``mobile_element_insertion`` step takes as the input the results of the ``ngs_mapping`` step
+(aligned reads in BAM format) and performs germline mobile element insertion identification (MEI).
+The result are tabular files with mobile insertion characteristics (txt files).
+
+==========
+Stability
+==========
+
+This step is considered experimental, use it at your own discretion.
+
+==========
+Step Input
+==========
+
+MEI identification step uses Snakemake sub workflows for using the
+result of the ``ngs_mapping`` step.
+
+===========
+Step Output
+===========
+
+For all samples, MEI identification will be performed on the primary DNA NGS libraries separately
+for each configured read mapper and mobile element identification tool. The name of the primary DNA
+NGS library will be used as an identification token in the output file.
+
+For each read mapper, MEI tool, and sample, the following files will be generated:
+
```

Comment:
Can we convert the output to VCF?

Reply:
I will put this change on hold. `scramble` has some internal issue with a non informative error message when I ask to output VCF...
**command:**
```
scramble.sh --ref /fast/projects/cubit/20.05/static_data/reference/GRCh37/hs37d5/hs37d5.fa \
			--out-name SAMPLE_PREFIX \
			--cluster-file /path/to/SAMPLE_cluster.txt \
			--nCluster 5 \
			--mei-score 50 \
			--indel-score 80 \
			--poly-a-frac 0.75 \
			--eval-meis
```
**error:**
```
Done analyzing l1 
Done analyzing sva 
Done analyzing alu 
Done analyzing l1 
Done analyzing sva 
Done analyzing alu 
Sample had 137 MEI(s)
Done analyzing MEIs
Loading required package: GenomicRanges
Error: subscript contains invalid names
Execution halted
```

Reply:
Done.